### PR TITLE
added logout link + change path to '/login' upon arrival

### PR DESCRIPTION
**Pull request by dan1212111**
Added the **logout** link.

### CODE REVIEW

**Review by phillipdunn — APPROVED** [review]
LGTM- may be dont need the extra white space though?

File: src/App.js [hunk 1]
```diff
@@ -3,11 +3,14 @@ import LoginPage from "./components/users/login/LoginPage";
...
 import PostsPage from "./components/posts/PostsPage";
 import ProfilePage from "./components/users/userProfile/UserProfile";
-import { useState } from "react";
+import { useEffect, useState } from "react";
 import { Routes, Route, Navigate, Outlet } from "react-router-dom";
 
+
 function App() {
   const [role, setRole] = useState("");
+
```
Comment (phillipdunn) [inline src/App.js:12]: this space needed?

Comment (dan1212111) [reply src/App.js:11]: Resolved

**Review by KinTale — APPROVED** [review]
LGTM!

**Review by KinTale — DISMISSED** [review]

**Review by nicopicchio — APPROVED** [review]
LGTM

**Comment by mikemherron** [comment]
I think this still has these issues:

```
The isLoggedIn method is checking if the "token" item in local storage is an empty string. I think the value we use for token is "jwt", not "token" - but also ,if it's not set at all, it will be undefined rather than an empty string. The logout function might set it to an empty string but we need to handle both cases. 
``

**Review by usamalodhi — COMMENTED** [review]
I approve these changes

**Review by usamalodhi — APPROVED** [review]
I approve these changes

**Pull request merged**